integration/kubernetes/helm-chart/alluxio/templates/master/service.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+kind: Service
```

Comment:
same here: changes from @jiacheliu3's embedded journal PR (https://github.com/Alluxio/alluxio/pull/9832/files) have been lost

Reply:
we can merge in a separate PR as well